sympy/physics/control/lti.py

```diff
@@ -526,6 +562,12 @@ def is_biproper(self):
         """
         return degree(self.num, self.var) == degree(self.den, self.var)
 
+    def _to_expr(self):
+        """
+        To convert TransferFunction type to SymPy Expr
+        """
```

Comment:
Make this one line?

Reply:
Why private though? This could be a way for users to convert a transfer function to an elementary function.

Reply:
> Why private though? This could be a way for users to convert a transfer function to an elementary function.

Since the only possible use case, in my view at the time of its implementation, was the internal conversion of `TransferFunction` objects to `Expr` to make them compatible with the native matrix operations (since non-expr operations were deprecated).
If there are other possible use cases that a user might find helpful, I will make it a public method. I will do whatever you suggest.

Reply:
```python
>>> tf = TransferFunction(s + 1, s**2 + 2, s)
>>> delay = exp(-s/tau)
>>> expr = delay*tf.to_expr()
>>> H1 = TransferFunction.from_rational_expression(expr, s)
>>> 
>>> H2 = TransferFunction(exp(-s/tau)*(s + 1), s**2 + 2, s)
```

Both `H1` and `H2` should work, I guess.

Reply:
> Both `H1` and `H2` should work, I guess.

This example looks fine but in the call which we had, @moorepants suggested not to implement `from_rational_expression()` classmethod.

Reply:
> ```python
> >>> tf = TransferFunction(s + 1, s**2 + 2, s)
> >>> delay = exp(-s/tau)
> >>> expr = delay*tf.to_expr()
> >>> H1 = TransferFunction.from_rational_expression(expr, s)
> >>> 
> >>> H2 = TransferFunction(exp(-s/tau)*(s + 1), s**2 + 2, s)
> ```

I think it's alright to go with that. I see similar classmethods being used in the codebase. 

There's a `to_expr` method in _holonomic.py_ that converts a holonomic function back to elementary function:
https://github.com/sympy/sympy/blob/8b783d15caae900e05c9236bf001fde908befe7a/sympy/holonomic/holonomic.py#L2066

It would be nice if ours did too. This PR is almost ready after that.

Reply:
I will complete the docstrings and tests for `from_rational_expression` by today. Hopefully, we can merge this PR after that